s3/replication/common/src/s3replicationcommon/s3_head_object.py

```diff
@@ -42,9 +42,357 @@ def __init__(self, session, request_id,
         self._timer = Timer()
         self._state = S3RequestState.INITIALISED
 
-    def get_contentlength(self):
-        """Returns content length for object."""
-        return int(self._response_headers["Content-Length"])
+    def get_accept_ranges(self):
+        """Get range of bytes for object.
+
+        Returns
+        -------
```

Comment:
please use google format of doc string. Reference : https://realpython.com/documenting-python-code/#google-docstrings-example

applicable to this complete file.

Reply:
Google format of docstring is not working. Codacy is showing an error for google docstrings. So I have kept the original docstring format.